receiver/prometheusreceiver/README.md

```diff
@@ -78,4 +78,33 @@ receivers:
               action: keep
 ```
 
+you can alternatively pass in the Prometheus configuration from a file, by setting `"config_file"` in the Prometheus Receiver configuration
```

Comment:
One of the motivations for having a separate file is that we **don't** do env var substitution on $ in the config.  I'd suggest:

1. We should document why someone may want a separate config file (e.g. configmap size limits, escaping $)
2. We should extend the examples (here and above) to use $ and $$ respectively.